Documentation/Getting-Started-For-Windows.md

```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```

Comment:
I don't know. I don't know what issues he was seeing when he didn't clone corefx. In the lab, and locally for me, this was not necessary.